src/main/java/berlin/yuna/nano/services/http/model/HttpObject.java

```diff
@@ -41,7 +50,20 @@ public static TypeMap convertHeaders(final Headers headers) {
             ));
     }
 
-    public static boolean isMethod(final HttpRequest request, final HttpMethod method) {
+    public static TypeMap convertSimpleHeaders(final Map<String, String> headers) {
```

Comment:
What does this method? `split()` is really expensive, specially using regex!
This method is not test covered